metacat-common-server/src/main/java/com/netflix/metacat/common/server/connectors/ConnectorContext.java

```diff
@@ -30,6 +30,14 @@
      * Catalog name.
      */
     private final String catalogName;
+    /**
+     * Catalog shard name.
+     */
+    private final String catalogShardName;
+    /**
+     * Catalog type.
+     */
+    private final String type;
```

Comment:
We should name the type explicitly as connectorType.

Reply:
Will do.